### ISSUE TITLE
Question about Fine tuning Model with 300W-LP

### ISSUE THREAD

**Issue by RookieDookie123** [vitoralbiero/img2pose#68]
Hi @vitoralbiero ,

I have read the Section 4.2 stated in the paper but I still can't understand how should I fine-tune my model with the dataset after training it with WIDER FACE. 
Could you tell me the steps on fine-tuning the model ?

Thank you in advance. Your helps are much appreciated.

**Comment by vitoralbiero** [comment]
Hello @RookieDookie123,

Thanks for your interest in our paper. 
After training on WIDERFACE, load the trained weights, in [models.py](https://github.com/vitoralbiero/img2pose/blob/fd5473efb83d78c530afc7db12b7c2aa631ea5cb/models.py#L70) change ```rpn_batch_size_per_image``` to 2 (proposals), and ```box_detections_per_img``` to 4 (head samples). Then, fined tune with fixed learning rate (0.001) for 2 epochs.

I hope this helps.
